packages/shorebird_cli/lib/src/flutter_validation_mixin.dart

```diff
@@ -11,9 +12,20 @@ mixin FlutterValidationMixin on ShorebirdCommand {
     ))
         .flattened;
     if (validationIssues.isNotEmpty) {
+      logger.info(
+        yellow.wrap('!!! Potential issues found !!!\n'),
+      );
+
       for (final issue in validationIssues) {
         logger.info(issue.displayMessage);
       }
+
+      logger.info(
+        yellow.wrap(
+          '\nThese may cause serious issues with shorebird functionality if '
```

Comment:
Similar, we should just explain the issues.  We're talking to pretty sophisticated users I would think?

e.g. I'd expect a validator which just said:

Warning: INTERNET permission is missing from AndroidManifest.xml.  Shorebird code push requires internet to pull updates.

Reply:
https://pink.bikeshed.org/ :)

Reply:
Done